microceph/ceph/osd.go

```diff
@@ -979,3 +979,50 @@ func killOSD(osd int64) error {
 	}
 	return nil
 }
+
+func SetReplicationFactor(spec string, size int64) error {
+	ssize := fmt.Sprintf("%d", size)
+	if spec == "" {
```

Comment:
But set the default to what? To the value the user set? It can be any arbitrary value, and setting it globally seems like a bad idea to me.

Reply:
@lmlg Indeed I was suggesting to set the user-provided target RF value as the new global default, basically the thing that charm-microceph does today when setting RF=1

```
["ceph", "config", "set", "global", "mon_allow_pool_size_one", "true"],
["ceph", "config", "set", "global", "osd_pool_default_size", "1"], 
```

I'm afraid this is necessary so that auto-created pools like `.mgr`, the RGW pools, the pools created by apps like nova/cinder etc. are getting the RF the user wants. I agree that it could be dangerous -- but IMHO making this an explicit action the user has to take lowers the risk this would slip in by accident.